vaadin-messages-flow-parent/vaadin-messages-testbench/src/main/java/com/vaadin/flow/component/messages/testbench/MessageElement.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.messages.testbench;
+
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+
+/**
+ * A TestBench element representing a <code>&lt;vaadin-message&gt;</code>
+ * element.
+ *
+ * @author Vaadin Ltd.
+ */
+@Element("vaadin-message")
+public class MessageElement extends TestBenchElement {
+
+    /**
+     * Gets the text content of the message body.
+     *
+     * @return the text content of the message body
+     */
+    @Override
+    public String getText() {
+        return this.getPropertyString("textContent");
+    }
+
+    /**
+     * Gets the {@code time} property of this element.
+     *
+     * @return the {@code time} property
+     */
+    public String getTime() {
```

Comment:
We force using an `Instant` through the Java API. Should we force the response to be an instant of this method as well? Or is it better to be more native and return the type that `vaadin-message` gives (String)? Dunno.

Reply:
Usually the TestBench APIs are closer to the web component APIs, because TB operates in the browser.